### ISSUE TITLE
Show eslint rule name

### ISSUE THREAD

**Issue by lukeed** [zed-industries/zed#4388]
### Check for existing issues

- [X] Completed

### Describe the feature

The tooltip message (or status message) should say what the ESLint rule name is that's failing. That way we can know which rule(s) to disable en masse.

Should also be listed in the Errors/Problems panel

### If applicable, add mockups / screenshots to help present your vision of the feature

<img width="466" alt="Screen Shot 2023-11-01 at 3 51 15 PM" src="https://github.com/zed-industries/community/assets/5855893/93f79a5b-5fa4-45fe-8680-c56dc30e3f16">

<img width="769" alt="Screen Shot 2023-11-01 at 3 51 57 PM" src="https://github.com/zed-industries/community/assets/5855893/b0792a67-e535-4d78-a5b3-f471b6fa2a3e">

<img width="415" alt="Screen Shot 2023-11-01 at 3 52 11 PM" src="https://github.com/zed-industries/community/assets/5855893/5e52075f-016f-435d-af7c-f5f5f5121083">

**Comment by shellscape** [comment]
Giving this some bump love.

**Comment by chadoh** [comment]
agree, almost makes it worth switching to another editor when trying to add linting to previously-unlinted files

**Comment by jaydenseric** [comment]
Also, the displayed rule name should be a link to the rule docs. I can't figure out in the current Zed UI for ESLint rule violations how to find a link to the rule.

**Comment by elyobo** [comment]
Maddening missing feature.

**Comment by JosephTLyons** [comment]
@mrnugget - might be a papercut you'd enjoy - I think you've done quite a bit of fixes in the js/ts dev world of Zed.

**Comment by OwsOwies** [comment]
For now, as a workaround you can `ctrl+shift+m` (Project diagnostics) and the rule is displayed there at the end 

![Image](https://github.com/user-attachments/assets/8196f63a-e194-4d91-96c0-d3399aad0eb9)

**Comment by nihgwu** [comment]
Fix https://github.com/zed-industries/zed/pull/28135 is waiting for review
